umbraco-cms/tutorials/editors-manual/working-with-content/rich-text-editor.md

```diff
@@ -30,211 +30,275 @@ To make your work easier, there are shortcut keys for certain editor functions.
 
 ### Adding a Macro to the Page
 
-1. Place the cursor in the editor where you want to insert your macro.
-2. Click the _**Insert Macro**_ button from the toolbar to open the macro properties dialog box.
-3. Select a macro to insert from the Choose a macro drop-down list.
-4. Configure the macro as required.
-5. Click _**Select**_ to insert the macro.
-6. The macro is displayed in the editor with a yellow dotted border.
+1. Place the cursor in the RTE where you want to insert your macro.
+2. Click the **Insert Macro** button from the toolbar.
+3. Select a macro to insert from the macros list.
+4. Click **Submit**.
 
-![macroPicker.jpg](images/macroPicker.jpg)
+   The macro is displayed in the editor with a dotted border.
 
-![macroEditor.jpg](images/macroEditor.jpg)
+![Adding a Macro to the Page](images/Select-Macro-v11.png)
 
 ### Editing macros
 
-You can edit values you have inputted into a macro after it has been added to the rich text editor.
+You can edit values after a macro has been added to the RTE:
 
-1. Highlight the macro in the rich text editor.
-2. Click the _**macro**_ button in the toolbar.
-3. This will bring out the configuration slide out menu where you can edit any values previously added.
+1. Select the macro in the RTE.
+2. Click the **Insert Macro** button from the toolbar.
+3. This will open the configuration menu where you can edit any values previously added.
 
 ### Removing macros
 
-To remove a macro you highlight the macro in the rich text editor and hit the _**delete**_ or _**backspace**_ button on your keyboard.
+To remove a macro:
+
+1. Select the macro in the RTE.
+2. Press the **Delete** or **Backspace** button on your keyboard.
 
 ## Tables
 
-Tables are used to format information in a grid based structure. When you insert a table, you select how many rows and columns the table should comprise of, as well as fill in a number of additional optional formatting properties. These values can be altered later, so it is not important to know exactly what your table will look like when you create it.
+Tables are used to format information in a grid-based structure. When you insert a table, you select how many rows and columns the table should comprise of. Additionally, you can fill in some optional formatting properties. These values can be changed later, so it is not important to know exactly what your table will look like when you create it.
 
-![tables.jpg](images/tables.jpg)
+![Inserting a Table](images/Insert-a-table-v11.png)
 
 ### Editing an Existing Table
 
-To edit the table after you have already created it, you need to click on it so you have a small square box in each of the four corners. Then you need to click on the _**Table**_ button in the rich text editor in order to reveal the options.
+To edit the table after creating it:
+
+1. Click on the table.
+2. A pop-up appears with different table properties and options.
+3. Alternatively, you can click on the **Table** button in the RTE toolbar.
```

Comment:
This list contains only a single action 🙈 
Does it make sense to convert it back again?

```suggestion
Select the table to open a pop-up with different table properties and options. Alternatively, you can click on the **Table** button in the RTE toolbar.
```